### PR TITLE
[#194] Add hours column to user tasks report

### LINKED ISSUE #194: Add hours column to user tasks report
[Trac import]
Reported by: jaragunde
Original date: Monday, 18 March 2013 10:02

A column named 'hours' would be convenient to complement the information in init and end time columns. Its value would just be (end time - init time ) / 60.0.

Consider hiding the init and end time columns in the standard view of the report once the hours column is added.

### PR DESCRIPTION
**Pull request by tonythomas01**
Fix #194

### CODE REVIEW

File: web/js/userTasksReport.js [hunk 1]
```diff
@@ -527,46 +532,50 @@ Ext.onReady(function () {
...
         columnModel.setColumnWidth(2, 55);
-        columnModel.setColumnWidth(3, 200);
-        columnModel.setColumnWidth(4, 120);
-        columnModel.setColumnWidth(8, 120);
-        columnModel.setColumnWidth(9, 435);
+        columnModel.setColumnWidth(3, 55);
+        columnModel.setColumnWidth(4, 200);
+        columnModel.setColumnWidth(5, 120);
+        columnModel.setColumnWidth(6, 120);
+        columnModel.setColumnWidth(7, 435);
```
Comment (jaragunde) [inline web/js/userTasksReport.js]: Please double-check the column width values, there are some mistakes. For example, the description column used to be 435 pixels width, but now you assigned that value to the onsite column.

Comment (tonythomas01) [inline web/js/userTasksReport.js]: done.

**Comment by jaragunde** [comment]
In the standard view, "task type" column should be hidden, and "story" should be shown instead.

**Comment by jaragunde** [comment]
> In the standard view, "task type" column should be hidden, and "story" should be shown instead.

This was actually a regression introduced in d16c4fa8986585400db49adc97860359dc88d43b

**Pull request merged**